src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <>
+      {tabs.map(tab => (
+        <li
+          className={selectedTab.id === tab.id ? 'is-active' : ''}
```

Comment:
we usually use classnames package for such cases

Reply:
its just more convient for me to use ternari here, if we use it just once